extensions/amp-bind/0.1/test/test-bind-integration.js

```diff
@@ -146,62 +146,47 @@ describe.configure().retryOnSaucelabs().run('amp-bind', function() {
     });
   });
 
+  /**
+   * Combined test specs for amp-img
+   */
   describe('amp-img integration', () => {
-    it('should change src when the src attribute binding changes', () => {
-      const button = fixture.doc.getElementById('changeImgSrcButton');
+    it('should do amp-img integration', () => {
+      const changeImgSrcButton = fixture.doc
+          .getElementById('changeImgSrcButton');
+      const invalidSrcButton = fixture.doc.getElementById('invalidSrcButton');
+      const ftpSrcButton = fixture.doc.getElementById('ftpSrcButton');
+      const changeImgAltButton = fixture.doc
+          .getElementById('changeImgAltButton');
+      const changeImgDimensButton = fixture.doc
+          .getElementById('changeImgDimensButton');
       const img = fixture.doc.getElementById('image');
       expect(img.getAttribute('src')).to.equal('http://www.google.com/image1');
-      button.click();
+      expect(img.getAttribute('height')).to.equal('200');
+      expect(img.getAttribute('width')).to.equal('200');
+      changeImgSrcButton.click();
```

Comment:
For this I'm referring specifically to the buttons that mutate the image source. `changeImgSrcButton`, `ftpSrcButton`, and `invalidSrcButton` all change the `src` attribute of the image element. In order to test that these buttons do what we expect, we have to test them one by one and then check the result. If all three buttons are clicked at once, we can't know what the outcomes of each of the clicks is, just what the end result is.